interfaces/apparmor/backend_test.go

```diff
@@ -114,6 +114,20 @@ func (s *backendSuite) TestInstallingSnapWritesAndLoadsProfiles(c *C) {
 	})
 }
 
+func (s *backendSuite) TestInstallingSnapWithHookWritesAndLoadsProfiles(c *C) {
+	devMode := false
+	s.InstallSnap(c, devMode, backendtest.HookYaml, 1)
+	profile := filepath.Join(dirs.SnapAppArmorDir, "snap.foo.hook.test-hook")
+
+	// Verify that profile "snap.foo.hook.test-hook" was created
+	_, err := os.Stat(profile)
```

Comment:
Yeah using `os.Stat()` here keeps the tests consistent with the rest of the suite.